3-professional-development/goal-setting.md

```diff
@@ -0,0 +1,57 @@
+
+# Application Engineering Goal Setting Overview
+
+In collaboration with their manager, engineers may lay out specific, measurable, and reachable goals on a 6 or 12 month repeating cadence, with a brief check-ins halfway through the cycle
+
+## Why do goal setting?
+
+When done well, this goal setting process serves a few purposes. First and foremost, it provides a framework for engineers to communicate their professional development goals to the managers so that those goals can be factored into work planning. It also creates a positive feedback loop for engineers as they track themselves getting closer to their goals. In our day-to-day, it's easy to lose sight of our medium and long term goals, especially when a project is in crunch time. Setting measurable goals on a rolling cadence helps to combat this by creating a sense of urgency around making steady, incremental progress. Goal setting is also a tool for managers. The outcomes from tracking these goals can be used as a resource for managers when performing the agency-wide review process and when making the case to leadership for promotions.
+
+## Timeline
+
+Engineers can choose to do their goal setting on a 6 month or 12 month cycle. 12 month cycles start in January and 6 month cycles will start in January and July. To kick off a cycle, engineers and their managers will hold two one on one meetings, roughly a week apart. A couple days prior to the first one on one, engineers will email a handful (very roughly 2-5) goal statements to their manager, along with any thoughts on them that they want to share. You can think of this as a "rough draft" of your goals. **To be clear, engineers are expected to send this rough draft to their manager at least two working days before the first meeting. This is so that the manager has time to read them and prepare feedback.** In the first meeting, the engineer and manager discuss the goals and the manager provides feedback. The manager will likely ask questions like "_why_ did you pick this goal" to better understand the underlying intention, so be prepared to answer those questions. The week between meetings is time for the engineer to consider any feedback they were given and refine their goals. The second meeting is for the engineer and manager to review the refined goals and mutually "sign off" on them for the engineer to get started. Once everyone feels good about the goals, your manager will share a document stating the goals for both parties to sign as recognition of the goals.
+
+Halfway through the cycle, engineers and managers will hold a brief "check-in" meeting to review progress on their goals. They may also decide to update or tweak goals to better fit the realities that are making the original goals difficult or impossible to achieve.
+
+For the rest of this document, it can be helpful to preface any example goal statements you see with "In the next 6/12 months, (I will...)".
```

Comment:
From:

> For the rest of this document, it can be helpful to preface any example goal statements you see with "In the next 6/12 months, (I will...)".

To:

> For the rest of this document, it can be helpful to preface any example goal statements with "In the next 6/12 months, (I will...)".

(removing "you")